types/proper-lockfile/index.d.ts

```diff
@@ -1,13 +1,16 @@
-// Type definitions for proper-lockfile 3.0
+// Type definitions for proper-lockfile 4.1
 // Project: https://github.com/moxystudio/node-proper-lockfile
 // Definitions by: Nikita Volodin <https://github.com/qlonik>
 //                 Linus Unnebäck <https://github.com/LinusU>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { TimeoutsOptions } from "retry";
```

Comment:
This defines the dependency on `retry` package. Lets add `package.json` with dependency on `retry@0.12`. I think it is described here: https://github.com/DefinitelyTyped/DefinitelyTyped#i-notice-some-packages-having-a-packagejson-here

Reply:
Thanks for info. Done.

Reply:
Okay. Removed it again because I got ...

> If you are depending on another `@types` package, do *not* add it to a `package.json`. Path mapping should make the import work.

See https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/531298518?utm_source=github_status&utm_medium=notification.

Reply:
Weird error. Seems like the readme is missing some details on when to include and when not to include dependencies.

Reply:
We check all of DT together, so future changes to retry will now have to avoid breaking proper-lockfile. That's how we are able to avoid specifying a version in package.json.